match/lib/match/git_helper.rb

```diff
@@ -48,25 +46,19 @@ def self.match_version(workspace)
     end
 
     def self.commit_changes(path, message, git_url)
-      Dir.chdir(path) do
-        return if `git status`.include?("nothing to commit")
+      git = Git.open(path)
 
-        Encrypt.new.encrypt_repo(path: path, git_url: git_url)
-        File.write("match_version.txt", Match::VERSION) # unencrypted
+      # Avoid calling git.status if no branch exists
+      return unless git.current_branch.nil? or git.status.any?
```

Comment:
I'm checking whether git has a current_branch registered or not. Will receive the name of the branch if a commit has been made before, or nil if it's a newly created branch. The reason i'm doing this is because calling git.status on a new branch (without any commits) throws an error (due to a bug in ruby-git).